PolyEngine/Engine/Src/ConfigBase.hpp

```diff
@@ -5,25 +5,29 @@
 
 namespace Poly
 {
-	enum class eConfigLocation
+	/// <summary>Enum used to identify source location of an asset.
+	enum class eResourceSource
 	{
-		ENGINE,
-		GAME,
-		LOCAL
+		ENGINE, // engine resources location
+		GAME, // game resources location
+		USR_LOCAL, // user local space resources location
+		NONE, // default (often runtime directory) resources location, this option will not modify path to resource
+		_COUNT
```

Comment:
Copy-pasting my comment, since it got hidden:

Reply:
Wait, so where do `ENGINE` and `GAME` point to?
>Engine has it's own configuration. That is not part of the Game.

Well, I'm asking what's the reasoning here.

Reply:
e.g. Keeping one configuration of the engine for different projects (games) or acessing shaders taht are part of engine resources.